parquet/benches/arrow_writer.rs

```diff
@@ -96,6 +96,24 @@ fn create_string_bench_batch(
     )?)
 }
 
+fn create_string_view_bench_batch(
```

Comment:
I didn't see that this tested both BinaryView and Utf8View at first (the name caught me up)

```suggestion
fn create_string_and_binary_view_bench_batch(
```